src/components/ContentNode/CodeListing.vue

```diff
@@ -10,9 +10,12 @@
 
 <template>
   <div
+    tabindex="-1"
     class="code-listing"
-    :data-syntax="syntaxNameNormalized"
     :class="{ 'single-line': syntaxHighlightedLines.length === 1 }"
+    :data-syntax="syntaxNameNormalized"
+    @keydown.meta.a.prevent="selectCode"
```

Comment:
I don't think this approach will work by adding this listener to this specific element, because then it only works after you have clicked the code listing and it has focus as the target of the events from ctrl+a.